### DISCUSSION TITLE
Neos Smartcam - partial updates resets all settings

### DISCUSSION THREAD

**Discussion by Trevsweb** [themactep/thingino-firmware#377] [Q&A]
when I do a partial update, I like updating my cameras often, all the settings get reset. is this normal?
all the OSD and some network settings(ip address) get reset. Is there a better way to update that I don't need to keep sorting my cameras out?

Is there an easy way to backup and restore configs? i've found the save config in the streamer section but cant find a restore function
also would love an update button in the ui

**Comment by tERB00** [comment]
You can backup your .cfg files from /etc/ folder if you want and upload them back after update. But it is not recommended to update your camera too often unless something is broken in yours or there is a new feature that you absolutely need.
